src/main/app/claims/claimModelConverter.ts

```diff
@@ -14,12 +14,15 @@ export class ClaimModelConverter {
     this.convertClaimantDetails(draftClaim)
     this.convertDefendantDetails(draftClaim)
 
-    draftClaim.claimant.dateOfBirth = draftClaim.claimant.dateOfBirth.date.asString() as any
     draftClaim.reason = draftClaim.reason.reason as any
 
+    draftClaim['feeAmountInPennies'] = draftClaim.claimant.payment.amount
+
     draftClaim['payment'] = draftClaim.claimant.payment as any
     delete draftClaim.claimant.payment
 
+    draftClaim.amount['type'] = 'breakdown'
```

Comment:
don't we have any enum for that?

Reply:
nope its not an enum, easier to just hardcode it here I would say, we shouldn't ever have to look at it.